### DISCUSSION TITLE
Upgrade: Files involved when changing the domain name

### DISCUSSION THREAD

**Discussion by ubani** [modoboa/modoboa#2091] [Q&A]
What files do I need to keep in mind to change the domain name for my modoboa instance?

Thanks

**Comment by tonioo** [comment]
@ubani Those files at least:
* /etc/postfix/main.cf
* Files in /etc/amavis/
* /srv/modoboa/instance/instance/settings.py
* Files in /etc/nginx
* Files in /etc/dovecot
* /etc/radicale/config
* /etc/automx.conf
